docs/domains/Domain.json

```diff
@@ -271,59 +345,33 @@
           "description": "If present, every time this value is updated the operator will restart the required servers.",
           "type": "string"
         },
+        "introspectVersion": {
+          "description": "If present, every time this value is updated, the operator will start introspect domain job",
```

Comment:
I thought introspectVersion was removed.  Why is it showing up here?

@jshum2479

Reply:
If it must remain, can it's descriptions in all locations be decorated with something like 'This is an experimental setting that isn't supported in production deployments.'

@rjeberhard FYI